templates/types/streaming/fastapi/app/api/controllers/llama_cloud.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+from typing import Any, Dict, List, Optional
+
+import requests
+
+logger = logging.getLogger("uvicorn")
+
+
+class LLamaCloudFileController:
+
+    LLAMA_CLOUD_URL = "https://cloud.llamaindex.ai/api/v1"
+    LOCAL_STORE_PATH = "output/llamacloud"
+
+    DOWNLOAD_FILE_NAME_TPL = "{pipeline_id}${filename}"
+
+    @classmethod
+    def get_files(cls, pipeline_id: str) -> List[Dict[str, Any]]:
```

Comment:
mark private methods?

Reply:
It just class static method, no need to set as private

Reply:
isn't python usually marking `_` prefix for private stuff? (see https://peps.python.org/pep-0008/#method-names-and-instance-variables)

I know it's not really private in Python, but that way we mark what should be called and what not